### PR TITLE
Add javascript and python engines to wasm worker

### PR DESCRIPTION
**Pull request by tomasol**
Summary: Allow running javascript and python lambda tasks inside WebAssembly worker. WASM resources are taken from wapm.io repository, Wasmer runtime is installed inside the container.

Test Plan: Please follow readme.md where simple workflow with python and javascript tasks is created and executed.

### CODE REVIEW

**Comment by KenG98** [comment]
Really excited for this functionality.

I see you're still working on this (new commits coming in) so I'll give very high level feedback. We should avoid committing the python source. I think the approach here is to compile the python interpreter into WASM (which is good).

Instead of committing the Python source, could we do a `git clone` of the python source in the dockerfile during build? So the build would clone the python source and do any relevant wasm building?

Thanks

**Comment by KenG98** [comment]
Also regarding the 5x-20x slowdown of running python through wasm. I think we might get a major performance win by using a small subset of python. Using vanilla python includes a lot of stuff our users shouldn't be using, like the server library, some concurrency libraries, etc. There is probably some open source project which provides a very bare-bones python interpreter that would run faster in WASM.

For now let's stick to vanilla python as-is, and just keep in mind for when performance becomes an issue that that's an option.

**Comment by tomasol** [comment]
I've removed the `lib` folder from this PR and constructing it in Dockerfile as suggested. Apparently python requires this folder to be writable so I have to copy it to a temp directory for every execution - this adds more latency, currently it takes around **1s** to run a simple python script. I agree we should investigate projects like micropython or rustpython in the long term.

File: symphony/app/fbcnms-projects/workflows-proxy/src/proxy/utils.js [hunk 1]
```diff
@@ -42,7 +42,6 @@ const SYSTEM_TASK_TYPES: Array<string> = [
   'EXCLUSIVE_JOIN',
   'WAIT',
   'DYNAMIC',
-  'LAMBDA',
```
Comment (KenG98) [inline symphony/app/fbcnms-projects/workflows-proxy/src/proxy/utils.js:45]: Are the wasm workers ready enough that we can disable lambda right away?

**Review by KenG98 — COMMENTED** [review]
What are `python.wasm` and `quickjs.wasm` and where are they from? Can they also be generated / downloaded in the `docker build` step? I'd like to avoid committing large binaries if possible.

**Comment by tomasol** [comment]
I am building quickjs inside docker now, however due to an [open issue](https://github.com/wapm-packages/python/issues/1) I am just downloading python.wasm from its git repository.

File: symphony/app/fbcnms-projects/workflows-proxy/src/proxy/utils.js (hunk 1 above)
Comment (tomasol) [reply symphony/app/fbcnms-projects/workflows-proxy/src/proxy/utils.js:45]: It is up to discussion if we want to do it now, but there is no way to fix the security problems associated with lambdas.

**Pull request closed without merging**